Dfe.PrepareConversions/Dfe.PrepareConversions.Tests/Pages/TaskList/PreviewProjectTemplateTests.cs

```diff
@@ -64,4 +70,52 @@ void PostProjectSetup(AcademyConversionProject project)
       await OpenAndConfirmPathAsync($"/task-list/{project.Id}/preview-project-template");
       Document.QuerySelector("#rationale-for-project").Should().NotBeNull();
    }
+
+   [Fact]
+   public async Task Given_InvoluntaryConversion_And_No_AdvisoryBoardDate_When_Previewed_Then_ValidationError_Shown()
+   {
+      static void PostProjectSetup(AcademyConversionProject project)
+      {
+         project.AcademyTypeAndRoute = AcademyTypeAndRoutes.Sponsored;
+         project.HeadTeacherBoardDate = null;
+      }
+
+      AcademyConversionProject project = AddGetProject(PostProjectSetup);
+      await OpenAndConfirmPathAsync($"/task-list/{project.Id}/preview-project-template");
+      Document.QuerySelector("#error-summary-title").Should().NotBeNull();
+      Document.QuerySelector("#error-summary-title").Text().Trim().Should().Be("There is a problem");
+
+      string selector = "*[data-eltype='error-link']";
```

Comment:
What is eltype?

Reply:
it's an element type, used because the element being found has a dynamic id based on an error key which is awkward.
Will rename it to 'data-element-type' to be clearer.